### DISCUSSION TITLE
UI ColorPickerEditor by Estrusco giving me an error.

### DISCUSSION THREAD

**Discussion by imhemayatsangin** [serenity-is/Serenity#6149] [Help] [answered]
I am using the UI **ColorPickerEditor by Estrusco** and when i opening the form dialog where i have added the color picker it shows me the following error.
SCRIPT ERROR! See browser console (F12) for details.
Message: Uncaught TypeError: $(...).colorpicker is not a function

File: http://localhost:59952/BedStatus/BedType#, Line: 1, Column: 49

Please if anybody can help me out what is the problem, Im using .Net core 5 and i have added the js and css files in the _layouthead.cshtml as per the guide. Please help me.

**Comment by wezmag** [comment, marked as answer]
Please check if the js and css files are loaded correctly using Network tab in F12 Developer Tools.

**Reply by imhemayatsangin** [reply]
Yep thanks! i added the following code in the BedTypeindex.cshtml and the color picker is now working but only one issue that when i edit the entry next time then it doesn't retrieve that color.
```
@{
    ViewData["Title"] = Serenity.LocalText.Get("Db.SomeEntity.EntityPlural");
}

<div id="GridDiv"></div>

@Html.Stylesheet("~/Scripts/colorpicker/bootstrap-colorpicker.min.css")
@Html.Script("~/Scripts/colorpicker/bootstrap-colorpicker.min.js")

<script type="text/javascript">
    jQuery(function () {
        new Serenity.SomeEntityGrid($('#GridDiv'), {}).init();

        Q.initFullHeightGridPage($('#GridDiv'));
    });
</script>
```
